addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -16,7 +16,7 @@
 }
 
 // Cards
-:where(.card:not([data-vcss])) .card-body {
+:where(.card:not([data-vxml])) .card-body {
```

Comment:
To whoever sees this: as discussed, we'll get rid of this rule somehow, before the release, I have to think about it :+1: